### PR TITLE
Adding play_mp3 action documentation to dfplayer component

### PR DESCRIPTION
**Pull request by llluis**
## Description:

Documentation for a new action on dfplayer component: play_mp3

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#4758

## Checklist:

  - [X] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by nagyrobi** [comment]
If you name the file, say "0001hello.mp3", do you reference it in ESPHome as "0001hello.mp3" or "0001.mp3" ?

**Comment by llluis** [comment]
> If you name the file, say "0001hello.mp3", do you reference it in ESPHome as "0001hello.mp3" or "0001.mp3" ?

Always by the number only. Everything after the number is ignored. So you reference it only with 1 or 0001.

**Comment by nagyrobi** [comment]
That should be noted in the doc.

**Comment by llluis** [comment]
Added more details in the instructions.

File: components/dfplayer.rst [hunk 1]
```diff
@@ -111,6 +111,39 @@ Configuration options:
 - **loop** (*Optional*, boolean, :ref:`templatable <config-templatable>`): Repeats playing
   the same track. Defaults to ``false``.
 
+``dfplayer.play_mp3`` Action
+----------------------------
+
+Plays a track inside the folder ``mp3``. Files inside the folder must be numbered from 1 
+to 65535, like ``0001.mp3``, ``0002.mp3``, ... etc.
+The folder name needs to be ``mp3``, placed under the SD card root directory, and the 
+mp3 file name needs to be 4 digits, for example, "0001.mp3", placed under the mp3 folder.
```
Comment (jesserockz) [inline components/dfplayer.rst]: If the files must be 4 digits, then its not possible to have 65535, the max would be 9999

**Review by jesserockz — APPROVED** [review]

**Pull request merged**